rfcs/0098-community-team.md

```diff
@@ -0,0 +1,194 @@
+
+To those who might oppose any public statement of values: Isn't it better to have it up front, so you don't have to wonder?
+
+To those who might say that any statement of values is merely performative and that the proof is in actions, not words: These authors fully agree, but we're saying the words anyway. Please hold us accountable for living up to them.
+
+## How new members join and leave the community team
+[joining-and-leaving]: #joining-and-leaving
+
+1. New members may apply to join the community team, and existing team members will decide on these applications. The team will evaluate applicants by the length of time they've been part of the Nix community, by their good character, by their demonstrated ability to exercise good judgement, and by their commitment to public service and a willingness to be self-critical. To form the initial team, the founding members will be appointed by Graham at his sole discretion.
+
+  The community team may decide to reach out to active community members they think would be a good fit for the team, if the member has not already applied to the team. The community team should keep internal records of what such outreach has been done so as to not duplicate efforts or reach out to members who have expressed disinterest prior.
+
+  The team's focus should be on developing leadership from within, not on bringing in experienced people from outside. This is important to maintaining ideological continuity.
+
+2. Community team members may resign at any time. Such resignations are effective immediately. In exceptional circumstances, members may also be removed by decision of the team; it is hoped that this will never be needed.
+
+3. The community team will decide on new members by consensus. That is, new members will be added to the team if and only if every then-current team member is in favor of adding them. In the event of removing members, the team will decide by a 2/3 majority vote, with the permissible votes being "yes" and "no". All team members are required to participate in membership decisions.
+
+  The rationale for using consensus as the basis of adding members is that it will lead to conservative decisions, which is expected to help with the goal of ideological continuity. This will also make it harder for a hostile takeover to succeed. Removing members is slightly easier than adding them, for the same reasons.
+
+  The formal decision-making process is intended only as a way to resolve membership questions and other long-term questions about the direction of the team; it is neither required nor recommended for routine moderation decisions.
+
+4. The team should attempt to maintain a sufficient size such that there are enough members to cover all the community spaces without major gaps. On no account should the team ever be so large that members can't all know each other, or can't keep each other in the loop on decisions.
```

Comment:
Right, yes, it does. The intent was to not specify an exact number, because that will get out of date quickly as things change, but instead give the team (and all future teams) some advice on how to decide on a number. So basically find the balance between these concerns.

I was also intending that the second one (team members must all know each other) takes precedence.

We can probably clarify this in the wording.